src/console/c_cvars.cpp

```diff
@@ -1745,3 +1747,55 @@ CCMD (archivecvar)
 		}
 	}
 }
+
+void C_GrabCVarDefaults ()
+{
+	int lump, lastlump = 0;
+	int lumpversion, gamelastrunversion;
+	gamelastrunversion = atoi(LASTRUNVERSION);
+
+	while ((lump = Wads.FindLump("DEFCVARS", &lastlump)) != -1)
+	{
+		FScanner sc(lump);
+
+		sc.MustGetString();
+		if (!sc.Compare("version"))
+			sc.ScriptError("Must declare version for defcvars!");
```

Comment:
To avoid mandatory version specification, we can assume 218 if no `version` is given.

Reply:
That is indeed possible, but with this post [here](https://forum.zdoom.org/viewtopic.php?p=1126200#p1126200) it seems preferable to keep the requirement in place. I personally do not mind making this optional if indeed there is a consensus to do so, though.